docs/getting_started/install_ez_enterprise.md

```diff
@@ -1,19 +1,44 @@
 # Install eZ Enterprise
 
+This page describes how to gain access to commercial packages and security releases
+when installing eZ Platform Enterprise Edition and eZ Commerce.
+They are available via [updates.ez.no](https://updates.ez.no).
+
 !!! enterprise
 
     To install eZ Enterprise, follow the procedure described in [Install eZ Platform](install_ez_platform.md).
     Because you do not have access to the Enterprise GitHub repositories,
-    you need to [download an archive](install_ez_platform.md#a-download-ez-platform) from the Support Portal.
+    you need to [download an archive](install_ez_platform.md#a-download-ez-platform) from the Support Portal, or follow instructions below in order to use `composer create-project`.
+
+    When dealing with software from updates.ez.no, you need to take care of [channels](#select-correct-updatesezno-channels) and [authentication](#set-up-authentication-tokens):
+
+    ## Select correct updates.ez.no channel(s)
+
+    There are two available channels:
```

Comment:
not two, six, and we have a seventh I didn't mention as it's mainly for internal use _(bul_nightly)_.
So maybe skip `two` here and not be specific on count